src/fiatExchanges/FiatExchangeCurrency.tsx

```diff
@@ -0,0 +1,250 @@
+      </View>
+    </TouchableWithoutFeedback>
+  )
+}
+
+function FiatExchangeCurrency({ route, navigation }: Props) {
+  const { t } = useTranslation()
+  const { flow } = route.params
+
+  const [selectedCurrency, setSelectedCurrency] = useState<Currency>(Currency.Dollar)
+
+  const goToProvider = () => {
+    ValoraAnalytics.track(FiatExchangeEvents.cico_option_chosen, {
+      flow,
+      currency: selectedCurrency,
+    })
+    if (flow === FiatExchangeFlow.Spend) {
+      return navigate(Screens.BidaliScreen, {
+        currency: selectedCurrency,
+      })
+    }
+    navigate(Screens.FiatExchangeAmount, {
+      currency: selectedCurrency,
+      flow: flow === FiatExchangeFlow.CashIn ? CICOFlow.CashIn : CICOFlow.CashOut,
+    })
+  }
+
+  const onSelectCurrency = (currency: Currency) => () => setSelectedCurrency(currency)
+
+  // TODO(jwaterman): https://github.com/valora-inc/wallet/issues/2394
+
+  // const [isEducationDialogVisible, setEducationDialogVisible] = useState(false)
+  // const onPressInfoIcon = () => {
+  //   setEducationDialogVisible(true)
+  //   // ValoraAnalytics.track(
+  //   //   isCashIn ? FiatExchangeEvents.cico_add_funds_info : FiatExchangeEvents.cico_cash_out_info
+  //   // )
+  // }
+
+  // const onPressDismissEducationDialog = () => {
+  //   setEducationDialogVisible(false)
+  //   ValoraAnalytics.track(
+  //     isCashIn
+  //       ? FiatExchangeEvents.cico_add_funds_info_cancel
+  //       : FiatExchangeEvents.cico_cash_out_info_cancel
+  //   )
+  // }
+
+  return (
+    <SafeAreaView style={styles.content}>
+      <ScrollView style={styles.topContainer}>
+        <View style={styles.titleContainer}>
+          <Text style={styles.selectDigitalCurrency}>{t('selectDigitalCurrency')}</Text>
+          {
+            // TODO(jwaterman): https://github.com/valora-inc/wallet/issues/2394
+            // <Touchable onPress={onPressInfoIcon} hitSlop={variables.iconHitslop}>
+            //   <InfoIcon size={14} color={colors.gray3} />
+            // </Touchable>
+          }
+        </View>
+        <View style={styles.currenciesContainer}>
+          <CurrencyRadioItem
+            title={t('celoDollar')}
+            body="(cUSD)"
+            selected={selectedCurrency === Currency.Dollar}
+            onSelect={onSelectCurrency(Currency.Dollar)}
+            containerStyle={{
+              borderTopLeftRadius: 8,
+              borderTopRightRadius: 8,
+              borderBottomWidth: 0.5,
+            }}
+            testID="radio/cUSD"
+          />
+          <CurrencyRadioItem
+            title={t('celoEuro')}
+            body="(cEUR)"
+            selected={selectedCurrency === Currency.Euro}
+            onSelect={onSelectCurrency(Currency.Euro)}
+            containerStyle={{
+              borderTopWidth: 0.5,
+              borderBottomWidth: 0.5,
+            }}
+            testID="radio/cEUR"
+          />
+          <CurrencyRadioItem
```

Comment:
worth adding cREAL, maybe with a feature flag? We're going to have providers supporting them soon

Reply:
feel free to punt with an issue at your discretion

Reply:
Issue created https://app.zenhub.com/workspaces/acquisition-squad-sprint-board-6010683afabec1001a090887/issues/valora-inc/wallet/2450